packages/fetchai/protocols/ledger_api/protocol.yaml

```diff
@@ -18,3 +18,9 @@ fingerprint:
 fingerprint_ignore_patterns: []
 dependencies:
   protobuf: {}
+package_hash: QmaXuUtRZZKum4M6j8k2RdjuzDCNYbF4wMf8348fFYvSME
+ipfs:
+  connections: {}
+  protocols: {}
+  skills: {}
+  contracts: {}
```

Comment:
Here we should have the map from public id to ipfs hash of the dependency. The question is where they are kept